packer.mk

```diff
@@ -19,6 +19,9 @@ run-packer-windows: az-login
 az-copy: az-login
 	azcopy-preview copy "${OS_DISK_SAS}" "${SA_CONTAINER_URL}?${SA_TOKEN}" --overwrite=false
 
+new-az-copy: 
```

Comment:
could you please rename to `az-copy-rbac`? "new" does not explain what's going on

Reply:
Updated